src/model/props-context-model.tsx

```diff
@@ -1,3 +1,3 @@
-export interface PropsModel{
+export interface PropsContextModel{
```

Comment:
I think it doesn't make sense to separate out into a separate file a type that is actually just one-liner. I would leave declarations of this type in the context file. In the "model" folder, I would keep domain models such as product, basket, etc.

Reply:
at the moment where, i remove context, this model is not required